src/services/scene_manager.py

```diff
@@ -11,6 +11,7 @@
 from src.scenes.level_scene import LevelStatus, LevelScene
 from src.scenes.scene import Scene
 from src.scenes.start_scene import StartScene
+from src.gui.language import DATA_PATH
```

Comment:
That's the only change in this file, is it required?

Reply:
Nice catch. It is not required. I used to change the path of maps but regreted. This line is a left over.